cms/djangoapps/contentstore/xblock_storage_handlers/view_handlers.py

```diff
@@ -1071,6 +1071,8 @@ def create_xblock_info(  # lint-amnesty, pylint: disable=too-many-statements
                 "published": published,
                 "published_on": published_on,
                 "studio_url": xblock_studio_url(xblock, parent_xblock),
+                "lms_url": f"{settings.LMS_ROOT_URL}/courses/{course.id}/jump_to/{xblock_info['id']}",
```

Comment:
should we create a `xblock_lms_url`? similar tu the `xblock_studio_url`

Reply:
Maybe we could create two other functions for `lms_url` and `embed_lms_url`